code/modules/cm_aliens/structures/special/egg_morpher.dm

```diff
@@ -112,6 +124,29 @@
 	COOLDOWN_START(src, spawn_cooldown, get_egg_cooldown())
 	if(stored_huggers < huggers_to_grow_max)
 		stored_huggers = min(huggers_to_grow_max, stored_huggers + 1)
+		if(stored_huggers == 1)
+			deploy_egg_triggers()
+
+/obj/effect/alien/resin/special/eggmorph/proc/create_egg_triggers()
+	for(var/item in 1 to 8)
+		egg_triggers += new /obj/effect/egg_trigger(src, src)
+
+/obj/effect/alien/resin/special/eggmorph/proc/deploy_egg_triggers()
+	var/item = 1
+	var/x_coords = list(-1,-1,-1,0,0,1,1,1)
+	var/y_coords = list(1,0,-1,1,-1,1,0,-1)
+	var/turf/target_turf
+	for(var/trigger in egg_triggers)
+		var/obj/effect/egg_trigger/ET = trigger
+		target_turf = locate(x+x_coords[item],y+y_coords[item], z)
+		if(target_turf)
+			ET.forceMove(target_turf)
+			item++
```

Comment:
why not just coords = list(list(-1, 1), list(-1, 0), list(-1, -1), list(0, 1)... etc). feels more readable?